Loop/Managers/NightscoutDataManager.swift

```diff
@@ -138,7 +138,51 @@ final class NightscoutDataManager {
         let loopName = Bundle.main.bundleDisplayName
         let loopVersion = Bundle.main.shortVersionString
 
-        let loopStatus = LoopStatus(name: loopName, version: loopVersion, timestamp: statusTime, iob: iob, cob: cob, predicted: predicted, recommendedTempBasal: recommended, recommendedBolus: recommendedBolus, enacted: loopEnacted, failureReason: loopError)
+   //add new loop parameters
+        var loopParams : String = ""
+        var loopBGTlow : String = ""
+        var loopBGThigh : String = ""
+        var loopEventualBG : String = ""
+        if let loopBGRange = deviceDataManager.loopManager.settings.glucoseTargetRangeSchedule?.value(at: Date()), let userUnit = deviceDataManager.loopManager.settings.glucoseTargetRangeSchedule?.unit {
+            if userUnit == HKUnit.milligramsPerDeciliter() {
+                loopBGTlow = String(Int((loopBGRange.minValue)))
+                loopBGThigh = String(Int((loopBGRange.maxValue)))
+            }
+            else
+            {
+                loopBGTlow = String(format:"%.1f", (loopBGRange.minValue))
+                loopBGThigh = String(format:"%.1f",(loopBGRange.maxValue))
+            }
+        }
+        
+        else {
+            loopBGTlow = "N/A"
+            loopBGThigh = "N/A"
+        }
+        
+        if let loopEventualBGquantity = predictedGlucose?.last?.quantity, let userUnit = deviceDataManager.loopManager.settings.glucoseTargetRangeSchedule?.unit  {
+            if userUnit == HKUnit.milligramsPerDeciliter() {
+                loopEventualBG = String(Int(loopEventualBGquantity.doubleValue(for: userUnit)))
+            }
+            else
+            {
+                loopEventualBG = String(format:"%.1f",loopEventualBGquantity.doubleValue(for: userUnit))
+            }
+        }
+        else
+        {
+            loopEventualBG = "N/A"
+        }
+        
+        loopParams = "BGTargets (" + loopBGTlow + ":" + loopBGThigh + ") | EvBG " + loopEventualBG + " | " + loopName
+        
+        //upload loopParams instead of just loopName
+        //that is the only pill that has the option to modify the text
+        let loopStatus = LoopStatus(name: loopParams, version: loopVersion, timestamp: statusTime, iob: iob, cob: cob, predicted: predicted, recommendedTempBasal: recommended, recommendedBolus: recommendedBolus, enacted: loopEnacted, failureReason: loopError)
```

Comment:
Rather than overriding the "name" field, a new field/fields should be added to the upload structure in the rileylink_ios project.

Reply:
@ps2 I can do that no problem, but how will NS display it ?  I dont think NS will just create a new pill.  But I may be wrong on that!

Reply:
Yes, supporting this in NS would require an NS PR.

Reply:
@kenstack Is it a complicated thing?

Reply:
@Kdisimone @ps2 it is still possible to pull to the dev?